README.md

```diff
@@ -1,3 +1,51 @@
+# Forking metabase
+
+Here at Enki we decided to fork metbase for the given time period in order to fix and make use of better connections to MongoDB. The underlying issue is that post-setup SSL connections (from admin panel) to Atlas hosted MongoDB databases error with false positives. This issue was fixed in newer versions of the mongoDB driver (Metabases still uses `3.2`), but never got implemented in Metabase because they wait for it to be upgraded in [Monger](https://github.com/michaelklishin/monger)
+
+This version was forked at `v0.30.5` and consists of these updates:
+
+- [update](https://github.com/metabase/metabase/pull/8459) jdk and jre versions in docker builds 
+- fix frontend typo `database` vs `Database` (todo add PR link)
+- fix backend typo `password` vs `pass` (invalid credentials error, todo add PR)
+
+To update metabase from the forked version, one need to create a `.zip` file pointing to the most up to date docker image and upload it manually to the Elastic BeanStalk UI. Follow these steps:
+
+1. Build metabase docker image by running:
+
+```sh
+$ docker build -t enkidevs/metabase:0.30.7-SNAPSHOT .
+# wait for build
+```
+
+To test the update locally, one can run the previously built image:
+
+```sh
+$ docker run --rm -d -p 3000:3000/tcp enkidevs/metabase:0.30.7-SNAPSHOT
```

Comment:
I'd put the more verbose versions `--detach` and `--port` of the flags here maybe? Could also break it up into multiple lines.

Reply:
You can also assign `--name` here and then use that in the `docker logs` command bellow.